SharpBucket/V2/EndPoints/RepositoryResource.cs

```diff
@@ -220,19 +220,19 @@ public object GetCommitComment(string revision, int commentId)
         /// </summary>
         /// <param name="revision">The commit's SHA1.</param>
         /// <returns></returns>
-        public object ApproveCommit(string revision)
+        public void ApproveCommit(string revision)
```

Comment:
The approve commit return a Participant object
https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/commit/%7Bnode%7D/approve

I don't ask to write the Participant POCO in that pull request, but maybe we should not change from object to void since it should return something one day